bigchaindb/core.py

```diff
@@ -115,6 +117,17 @@ def info(self, request):
 
         self.abort_if_abci_chain_is_not_synced()
 
+        # Check if BigchainDB supports the Tendermint version
+        try:
+            if check_tendermint_version(request.version):
+                logger.info(f"Tendermint version: {request.version}")
+            else:
+                logger.error(f"Error: Tendermint version {request.version} not supported."
+                             f" Currently, BigchainDB only supports {__tm_supported_versions__} .Exiting!")
+                sys.exit(1)
+        except AttributeError:
+            logger.debug("Skipping Tendermint version check.")
```

Comment:
Maybe we should say _why_ the Tendermint version check is being skipped: apparently the `request` from Tendermint didn't include a `version` attribute.